spec/models/instrument_spec.rb

```diff
@@ -488,6 +488,15 @@
       assert_equal 45, @next_reservation.reserve_start_at.min
     end
 
+    it "should find next available reservation with cutoff hours" do
+      expect(@rule.instrument.update_attribute :cutoff_hours, 10).to be true
```

Comment:
Not a big deal, but I generally put these kind of setup steps outside an `expect`. I would consider putting it inside a `before` block.